blog/blog.html

```diff
@@ -66,7 +36,7 @@
     <nav>
       <div id="navbar" class="navbar" style="height: 63px">
         <a href="#home" id="logo" class="anchor"
-          ><img src="../photos/logo.png" alt="KyootyBella"
+          ><img src="../assets/logos/logo.png" alt="KyootyBella"
```

Comment:
I renamed your /photos/ folder to the standard: /assets/. The struc is now as follows: 

```
assets
    images    
    icons
    logos
```